documentation/asciidoc/accessories/iqaudio/configuration.adoc

```diff
@@ -18,66 +18,59 @@ Find the `dtparam=audio=on` line in the file and comment it out by placing a # s
-the recording of audio from the built in MEMS microphone, from stereo Phono sockets (AUX
-IN) or 2 x mono external Electret microphones. Playback is through stereo Phono sockets (AUX OUT)
+the recording of audio from the built-in MEMS microphone, from stereo phono sockets (AUX
+IN) or two mono external electret microphones. Playback is through stereo phono sockets (AUX OUT)
 or a mono speaker connector.
 
-Each input and output device has its own “mixer” allowing the audio levels and volume to be adjusted
-independently. Within the codec itself other mixers and switches exist to allow the output to be mixed to a single mono channel
-for single speaker output. Signals may also be inverted and there is a 5 band Equaliser to adjust
-certain frequency bands. These settings can be controlled interactively using Alsamixer (a graphical mixer program used in Terminal) or programmatically.
+Each input and output device has its own mixer, allowing the audio levels and volume to be adjusted
+independently. Within the codec itself, other mixers and switches exist to allow the output to be mixed to a single mono channel for single-speaker output. Signals may also be inverted; there is a five-band equaliser to adjust certain frequency bands. These settings can be controlled interactively, using AlsaMixer, or programmatically.
 
-It is important to note that the AUX IN and AUX OUT are both 1V RMS. It may be necessary to adjust
-the AUX IN’s mixer to ensure the input signal doesn’t saturate the ADC (Analogue to Digital
-Convertors). Similarly, the output mixers can be to be adjusted to get the best possible output.
+Both the AUX IN and AUX OUT are 1V RMS. It may be necessary to adjust
+the AUX IN’s mixer to ensure that the input signal doesn’t saturate the ADC. Similarly, the output mixers can be to be adjusted to get the best possible output.
 
-There is a set of preconfigured scripts (loadable ALSA settings) available on GitHub. https://github.com/iqaudio/Pi-Codec.
-
-These cover several use cases such as:
+Preconfigured scripts (loadable ALSA settings) are available on GitHub. https://github.com/iqaudio/Pi-Codec, offering:
  
 * Mono MEMS mic recording, mono speaker playback
 * Mono MEMS mic recording, mono AUX OUT playback
 * Stereo AUX IN recording, stereo AUX OUT playback
 * Stereo MIC1/MIC2 recording, stereo AUX OUT playback
 
-The Codec Zero DA7212 chip needs to know which of these input and output settings are for each new cycle of power for it to operate correctly. Using a Terminal session connected to your Raspberry Pi either directly using a keyboard, mouse and monitor or via a SSH (Secure Shell), run the following command to download the scripts:
+The Codec Zero needs to know which of these input and output settings are for each new cycle of power for it to operate correctly. Using a Terminal session on your Raspberry Pi, run the following command to download the scripts:
```

Comment:
I think the "are for each" part doesn't make sense here, so the first sentence could be reworded to something like
"The Codec Zero needs to know which of these input and output settings are being used each time the Raspberry Pi powers on." ?

Reply:
Fixed